cmd/commands/function_test.go

```diff
@@ -208,6 +208,57 @@ var _ = Describe("The riff function create command", func() {
 			Expect(stdout.String()).To(Equal(fnCreateDryRun))
 		})
 
+		It("should display the nondefault namespace in status hint", func() {
```

Comment:
this description looks identical to the next test although they test different conditions

Reply:
thanks, that's an error, fixing it now

Reply:
Fixed in 3ef092e9a52ba976d30db126cbdd5f8d8497f3e1